doc/api/cli.md

```diff
@@ -625,6 +625,15 @@ Path to the file used to store the persistent REPL history. The default path is
 `~/.node_repl_history`, which is overridden by this variable. Setting the value
 to an empty string (`''` or `' '`) disables persistent REPL history.
 
+### `NODE_V8_COVERAGE=dir`
+
+When set, Node.js will begin outputting [V8 JavaScript code coverage][] to the
```

Comment:
neither the link nor the addition here explain what to do with the output coverage file. Generally people just use the inspector UI. does the output file even have a governed format? do we have to semver major when v8 changes the format?

Reply:
> Generally people just use the inspector UI

I would argue it's very rare for people writing command line tools and modules to interact with the inspector, as evidenced by the fact that [nyc](https://www.npmjs.com/package/nyc) has over `31,000` dependents and continues to grow in popularity (getting downloaded over 400,000 times a week) despite native test coverage having been available for quite some time.

As the lead maintainer of nyc and Istanbul, I would love to move people away from using these libraries when writing Node.js applications, and towards V8's coverage -- but I think the burden of kicking off, and interacting with an inspector session is a usability rough-edge that will prevent adoption.

> do we have to semver major when v8 changes the format?

I think you're right that perhaps we should point users to a breakdown of what to inspect in the output format -- @schuay can you recommend a better document to link to.

The output format is standardized in V8, and I don't expect that there will be major breaking changes to it (and if there are it will be very rare).

Reply:
sorry I wasn't clear. I'm saying there need to be docs for the format available from the node docs, not that people should only use the inspector UI. the link you have doesn't give any docs on the format of the file.

On the semver-major front, I just want to make sure we're careful, if you think it's fine that's good enough for me.

Reply:
let's let @schuay or someone else on V8's side chime in too (@TimothyGu?), I think your concern is valid and it would be good to have someone speak to how stable the API is on the Chromium side of things.

Reply:
Sorry, I'm not too involved in the V8 development process… though it might be telling that the coverage API is already in the [RC](https://chromedevtools.github.io/devtools-protocol/1-3/Profiler) version of the devtools protocol without an Experimental flag on.

Reply:
Great to see this :+1: 

The inspector protocol documentation [here](https://chromedevtools.github.io/devtools-protocol/tot/Profiler#method-takePreciseCoverage) is the source of truth. In particular, `takePreciseCoverage` will return an array of `ScriptCoverage` [objects](https://chromedevtools.github.io/devtools-protocol/tot/Profiler#type-ScriptCoverage). 

I'm not sure what DevTools' policies on protocol changes are. You could try asking on their mailing list: https://groups.google.com/forum/#!forum/google-chrome-developer-tools.

Reply:
Generally we are very careful with changing CDP due to backwards compatibility concerns. Unless unavoidable, we will not change the coverage format. If that happens however, the CDP version would be bumped as well.